src/ui/map.js

```diff
@@ -37,22 +37,22 @@ define(function(require, exports, module) {
     this.configureMap = function(ev, config) {
       this.trigger('mapStarted', {});
       // if list or facets are emabled, give the map less space
-      var addition = 0;
-      if (config.facets) {
-        addition += 300;
-      }
-      if (config.list) {
-        addition += 300;
-      }
-
-      if (addition > 0) {
-        window.setTimeout(function() {
-          if (this.map) {
-            this.$node.css('left', '+=' + addition);
-            this.map.invalidateSize();
-          }
-        }.bind(this), 50);
-      }
+      // var addition = 0;
```

Comment:
That sounds like a good fix; add classes to the body when the panels are enabled, and use that to control the CSS.